### ISSUE TITLE
[BUG] Splitbar for resizable Splitter panes should be focusable and have aria-valuenow attribute

### ISSUE THREAD

**Issue by Dimitar-Goshev** [telerik/kendo-themes#6078]
## Bug Description
Splitbar for resizable Splitter panes should be focusable and allow resizing panes via keyboard.
The splitbar has `role` [separator](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/separator_role#focusable_separator) and a focusable separator should have `aria-valuenow` attribute according to [a11y spec](https://w3c.github.io/aria/#separator).

Note 1: additional attributes are recommended to be added to focusable separator - "The [aria-valuemin](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-valuemin) and [aria-valuemax](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-valuemax) should also be included if they aren't set to the default values of 0 and 100, respectively." In jQuery Slitter all three attributes are set.

Note 2: non-focusable separator (statis splitbar) should not have aria-valuenow, aria-valuemin and aria-valuemax.

Note 3: The `separator` element in a Splitter should have `aria-controls` attribute, pointing to the primary pane - [WindowSplitter pattern](https://www.w3.org/WAI/ARIA/apg/patterns/windowsplitter/)

## Steps to Reproduce
1. Run the following [Dojo snippet in fullscreen mode](https://runner.telerik.io/fullscreen/dGrDGujV) . It is based on https://github.com/telerik/kendo-themes/blob/develop/tests/splitter/splitter-splitbar.html , but it also has tabindex="0" on the first draggable splitbar.
2. Scan the page with axe-core DevTools extension in Chrome.

## Expected Behavior
Splitbar has tabindex and aria-valuenow attributes. No a11y errors.

## Actual Behavior
Required ARIA attribute must be provided.

## Screenshots/Videos
<img width="1095" height="597" alt="Image" src="https://github.com/user-attachments/assets/738c76b5-32ba-45b8-8d73-e617ca5c92c3" />

### Affected Theme(s)
- [x] All themes

### Affected Component(s)
Splitter

### Affected Suites

- [x] Kendo UI for jQuery
- [x] Kendo UI for Angular
- [x] Kendo UI for React
- [x] Kendo UI for Vue
- [x] Telerik UI for Blazor
- [x] Telerik UI for ASP.NET Core
- [x] Telerik UI for ASP.NET MVC

### Browser Information
- [x] All browsers

### Build System Information
- Package version: @progress/kendo-themes-html@14.3.0
